dashboard/app/models/follower.rb

```diff
@@ -62,11 +62,11 @@ def assign_script
     student_user.assign_script(section.script) if section.script
   end
 
-  after_destroy :remove_family_name, if: proc {DCDO.get('family-name-features', false)}
+  after_destroy :remove_family_name
   def remove_family_name
     # If the student is in zero sections, and has a family name set,
     # remove the family name.
-    if student_user.family_name && student_user.sections_as_student.empty?
+    if student_user&.family_name && student_user.sections_as_student.empty?
```

Comment:
A related test wouldn't pass without this check- it kept complaining about calling 'family_name' on a nil object.